ReleaseProcessAutomation/Scripting/MSBuildCallAndCommit.cs

```diff
@@ -57,7 +58,14 @@ public int CallMSBuildStepsAndCommit (MSBuildMode msBuildMode, SemanticVersion v
     if (string.IsNullOrEmpty(msBuildPath))
     {
       _log.Warning("No MSBuild Path specified in config, will continue without MSBuild");
-      _console.WriteLine("There was no MSBuildPath specified in the config, will continue without Invoking MSBuild");
+      _console.WriteLine("There was no MSBuildPath specified in the config\nWill continue without invoking MSBuild");
+      return -1;
+    }
+
+    if (!File.Exists(msBuildPath))
+    {
+      _log.Warning("The configured MSBuildpath does not exist");
```

Comment:
please use `.` / full-stop for sentences. Most log-messages are (should be) complete sentences.